packages/node-opcua-server/source/opcua_server.ts

```diff
@@ -3498,12 +3498,13 @@ export class OPCUAServer extends OPCUABaseServer {
 
     private createEndpoint(
         port1: number,
+        hostname: string | undefined,
```

Comment:
use a `host` parameter instead, to distinguish from "hostname", host will be used to identify on which external or internal facing network interface available on the computer we want the socket to be listening. if not specified it would default to 0.0.0.0 meaning that the socket will listen on all available network interface, which is also the current behavior.

Reply:
![image](https://github.com/node-opcua/node-opcua/assets/42715397/311f0426-9712-415c-917c-5b84df28a438)

Hi Etienne,

Thank you for providing such a detailed description of the problem. 
I implemented the changes you suggested.